include/nccl_ofi_tracepoint.h

```diff
@@ -103,6 +103,21 @@
 	NCCL_OFI_TRACE_FLUSH_NVTX(request, nccl_req); \
 } while(0)
 
+#define NCCL_OFI_TRACE_READ(request, nccl_req) do { \
```

Comment:
I don't see an argument why this should be in a previous commit, since read is only implemented in the second commit of the two commits of this PR.

Reply:
Then a separate commit.  Lay the groundwork, and then have the commit that wires everything together, so that it is minimally sized and has only the really fiddly bits in it.

Reply:
Moved the tracing code into a commit that is ontop of the others.